code/modules/ninja/suit/suit.dm

```diff
@@ -170,7 +152,7 @@ Contents:
 	..()
 	if(s_initialized)
 		if(user == affecting)
-			to_chat(user, "All systems operational. Current energy capacity: <B>[cell.charge]</B>.")
+			to_chat(user, "All systems operational. Current energy capacity: <B>[cell.maxcharge]</B>.")
```

Comment:
Is this change correct?

Reply:
Sure. The maximum charge is the capacity, right? The reason I did charge -> maxcharge instead of capacity -> charge is because the charge is already displayed live in the SpiderOS tab. I'll change it to be a percentage so it's consistent with the message when upgrading the cell.